Sources/ApolloTestSupport/MockGraphQLServer.swift

```diff
@@ -85,14 +90,26 @@ public class MockGraphQLServer {
       return expectation
     }
   }
+
+  public func expect<Operation: GraphQLOperation>(
+    _ operationType: Operation.Type,
+    file: StaticString = #filePath,
+    line: UInt = #line,
+    requestHandler: @escaping (HTTPRequest<Operation>) -> JSONObject
+  ) -> XCTestExpectation {
+    return self.expect(operationType, file: file, line: line) { request -> Result<JSONObject, Error> in
+      let result = requestHandler(request)
+      return .success(result)
+    }
+  }
```

Comment:
What's the thought on adding a synchronous version of this instead of just using the async version?

Reply:
Note that neither versions are asynchronous. The other one has been changed to return `Result`, but it's still synchronous.